NitroxLauncher/LauncherConfig.cs

```diff
@@ -57,6 +58,40 @@ public string SubnauticaPath
             }
         }
 
+        // Launch arguments used to launch Subnautica
+        private string subnauticaLaunchArguments = Settings.Default.LaunchArgs ?? "-vrmode none";
+        public string SubnauticaLaunchArguments
+        {
+            get => subnauticaLaunchArguments;
+            set
+            {
+                if (value != subnauticaLaunchArguments)
+                {
+                    subnauticaLaunchArguments = value;
+                    Settings.Default.LaunchArgs = value;
+                    Settings.Default.Save();
+                }
+                OnPropertyChanged();
```

Comment:
Shouldn't property changed also be skipped if `value` is the same as current value?